app/models/conversation-part.js

```diff
@@ -0,0 +1,11 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+import { belongsTo } from 'ember-data/relationships';
+
+export default Model.extend({
+  body: attr(),
+  readAt: attr(),
+
+  author: belongsTo('user', { async: true }),
```

Comment:
I think async: true is default...but I also like the explicitness...

Reply:
It wasn't always, and I was about to ask if it is now finally the default.